pwndbg/commands/argv.py

```diff
@@ -36,17 +36,16 @@ def argv(i=None):
 
     pwndbg.commands.telescope.telescope(start, n)
 
-@pwndbg.commands.Command
-@pwndbg.commands.OnlyWhenRunning
-def args():
-    """
-    Prints out the contents of argv.
-    """
-    argv()
 
-@pwndbg.commands.Command
+parser = argparse.ArgumentParser()
+parser.description = "Prints out the contents of the environment."
+parser.add_argument("name", nargs='?', type=str, default=None, help="Name of the environment variable to see.")
+@pwndbg.commands.ArgparsedCommand(parser,aliases=["env","environ"])
 @pwndbg.commands.OnlyWhenRunning
 def envp(name=None):
+    if name is not None:
+        gdb.execute('p $environ("%s")' % name)
+        return
```

Comment:
We should delete the leftover function docstring that is below.

Reply:
Offtopic comment: the python api's `environ` function has meh error handling:
```
pwndbg> p $environ("PWD=")
Python Exception <class 'TypeError'> Could not convert Python object: void.:
Error while executing Python code.
```
We might want to send a patch to GDB to fix that :P